bigtable/use-cases/fraud-detection/terraform/scripts/run_streaming_pipeline.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+PROJECT_ID=$1
+REGION=$2
+GCS_BUCKET=$3
+PUBSUB_INPUT_TOPIC=$4
+PUBSUB_OUTPUT_TOPIC=$5
+CBT_INSTANCE=$6
+CBT_TABLE=$7
+
+ML_ENDPOINT_ID=$(cat Scripts/ENDPOINT_ID.output)
+
+if { [ -z "$ML_ENDPOINT_ID" ]; }; then
+	echo "ML endpoint was not written properly."
+	exit 1
+fi
+
+echo "PROJECT_ID = $PROJECT_ID"
+echo "REGION = $REGION"
+echo "GCS_BUCKET = $GCS_BUCKET"
+echo "PUBSUB_INPUT_TOPIC = $PUBSUB_INPUT_TOPIC"
+echo "PUBSUB_OUTPUT_TOPIC = $PUBSUB_OUTPUT_TOPIC"
+echo "CBT_INSTANCE = $CBT_INSTANCE"
+echo "CBT_TABLE = $CBT_TABLE"
+echo "RANDOM_ID = $RANDOM_ID"
+echo "ML_ENDPOINT_ID = $ML_ENDPOINT_ID"
+
+mvn compile exec:java -f ../pom.xml -Dexec.mainClass=com.example.dataflowpipelines.FraudDetection -Dexec.cleanupDaemonThreads=false \
```

Comment:
Let's try to investigate whether there is a way to create a pipeline in Terraform and point it to Java artefact built by Maven. Ideally we should use Maven to build code jars, not infrastructure.